pkg/validator/types.go

```diff
@@ -79,4 +79,6 @@ const (
 
 	nameLimit       = "When using the ClusterRole naming scheme, the total length of KD cluster name + any of its role IDs must not exceed 56 characters."
 	badNamingScheme = "Naming scheme not valid in the config file."
+
+	maxMemberLimit = "Maximum number of total members per KD cluster supported is 1000"
```

Comment:
One tweak. Let's make maxMemberLimit have a `%d` token instead of hardcoding 1000 there. Then the code in validateCardinality can use fmt.Sprintf to compose the actual error message by substituting in the value of maxKDMembers.